index.html

```diff
@@ -5,22 +5,93 @@
+					</ul>
+				</nav>
+			</div>
+		</header>
+
+
+			
+<section id="main">	
+	<div class="container">	
+ 		<section id="middle">
+			<article class="content">
+
+
+					<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+					<img src="images/blog_photo1.jpg"></a>
+					<p><span class="firstcharacter">L</span>orem ipsum dolor sit amet, ad tale iriure invidunt eam, quo appetere complectitur id. Mel elitr accusamus eu. Saepe forensibus temporibus duo eu, nemore dicunt periculis ad sit, ubique putant aliquip ad sea. Nec in liber dolor.Per hinc inermis ocurreret ut. Sanctus corpora ne sed, in tempor iriure malorum vix, dico hinc corrumpit ut eum. Numquam efficiendi no vis, te sit modus viderer. Et oporteat facilisis vis, id vel dicat liber senserit.Ad usu graeci timeam democritum, eu est adhuc melius timeam. Cum et ornatus dolorum perfecto, mentitum vituperata adversarium ei vix. Nisl choro omnium duo id, ei cum possim tritani consectetuer, ea quaeque denique nam. No pro vero causae accusam, dicam dolorem nam an, nec ut commune epicurei adolescens. Ut cum eruditi eligendi, id quando labores eum. Vim at melius eligendi expetendis, pri eu choro inciderint.
+					</p>
+					<p class="more"><a href="#">Read More</a></p>
+			</article>
+					
+			<article class="content">
+
+					<h2>I traveled across the globe thanks to Relaxr's Excel automation</h2>
+					<img src="images/blog_photo2.jpg"></a>
+					<p><span class="firstcharacter">L</span>oremLorem ipsum dolor sit amet, ad tale iriure invidunt eam, quo appetere complectitur id. Mel elitr accusamus eu. Saepe forensibus temporibus duo eu, nemore dicunt periculis ad sit, ubique putant aliquip ad sea. Nec in liber dolor.Per hinc inermis ocurreret ut. Sanctus corpora ne sed, in tempor iriure malorum vix, dico hinc corrumpit ut eum. Numquam efficiendi no vis, te sit modus viderer. Et oporteat facilisis vis, id vel dicat liber senserit.Ad usu graeci timeam democritum, eu est adhuc melius timeam. Cum et ornatus dolorum perfecto, mentitum vituperata adversarium ei vix. Nisl choro omnium duo id, ei cum possim tritani consectetuer, ea quaeque denique nam. No pro vero causae accusam, dicam dolorem nam an, nec ut commune epicurei adolescens. Ut cum eruditi eligendi, id quando labores eum. Vim at melius eligendi expetendis, pri eu choro inciderint.
+					</p>
+					<p class="more"><a href="#">Read More</a></p>
+			</article>
+		</section>
+	
+
+		<section id="sidebar">
+			<h3>Categories</h3>
+				<ul>
+					<li><a href="#">Success stories</a></li>
+					<li><a href="#">Stats</a></li>
+					<li><a href="#">How tos</a></li>
+					<li><a href="#">Best of</a></li>
+					<li><a href="#">Business tips</a></li>
+				</ul>
+			<h3>About Relaxr</h3>
+			<p>Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.
+			<br>
+			<a href="#" class="learnmore">Learn More</a></p>
+			<img src="images/ad-unit.png" alt="Ad Unit"></a>
```

Comment:
You have an extra </a> tag after the <img>. Be sure to validate your HTML before turning in your homework!